### DISCUSSION TITLE
libc++abi: terminating with uncaught exception of type std::runtime_error: fopen() failed

### DISCUSSION THREAD

**Discussion by Bingo821227** [madMAx43v3r/chia-plotter#369] [Q&A]
libc++abi: terminating with uncaught exception of type std::runtime_error: fopen() failed
zsh: abort 

mac mini m1 16G 2T

**Comment by madMAx43v3r** [comment]
increase max open file limit

**Comment by madMAx43v3r** [comment]
`ulimit -n 3000`

**Comment by ggulik** [comment]
I'm replotting on an M1 mac and when using ThunderBolt attached SSD drive as my temp drive I'm getting the following error while creating a plot:

```
.
.
[P1] Table 1 took 189.895 sec
Progress update: 0.01
STDERR: libc++abi: terminating with uncaught exception of type std::runtime_error: thread failed with: fopen() failed with: No such file or directory
```

I do NOT get this error when using a physical drive.
The SSD drive seems to work fine otherwise.  I've already reformatted the SSD drive, still the errors persist.
